### ISSUE TITLE
Write a Script to “Create Changelog PR” Script

### ISSUE THREAD

**Issue by emilyrohrbough** [cypress-io/cypress#25158]
@emilyrohrbough commented on [Tue Sep 20 2022](https://github.com/cypress-io/cypress-internal/issues/485)

- Collect all “UNRELEASED” entires from CHANGELOG.md.
- Clone cypress-documentation repository
- Create `x.x.x-release` branch  & push if it does not exist
- Create new file `./content/_changeslogs/x.x.x.md` and add collected unreleased entries
- Commit changelog.
- Open new PR for `x.x.x-release` against `master` branch in cypress-documentation repository.

**Comment by emilyrohrbough** [comment]
Blocked by https://github.com/cypress-io/cypress/issues/25154.

**Comment by emilyrohrbough** [comment]
FYI - we should be able to update/use some of this existing script for some of this: https://github.com/cypress-io/release-automations/tree/master/packages/set-releases
